### DISCUSSION TITLE
Dependencies from submission-api are not shown in organization dependency insights

### DISCUSSION THREAD

**Discussion by cmergenthaler** [community/community#58884] [Actions]
### Select Topic Area

Bug

### Body

On repository views all dependencies are shown in the graph but when we want to have an overview over the used dependencies in our github organization, the submitted dependencies with submission-api are not shown on the organization wide dependency insights view. 

Is this known/expected? I've considered this as a bug because the views are not consistent.

**Comment by cmergenthaler** [comment]
Any statement on this?

**Comment by MPV** [comment]
I think this works now, we are seeing it since a few weeks back. Do you confirm that you're also seeing it? Then I guess this one could be closed?

**Comment by cmergenthaler** [comment]
@MPV We are still not seeing any dependencies that are pushed via the dependency submission api on organization level.

**Reply by MPV** [reply]
Oh, my bad, you are right, we are actually not yet seeing this either. 

My mixup was due to Dependabot Alerts (also on org level) recently started supporting vulnerabilities found based on data uploaded to the Dependency Submission API.
